src/utils/batt_utils.ts

```
@@ -0,0 +1,231 @@
+        output = execSync(scriptCommand, {
+          encoding: 'utf8',
+          maxBuffer: 1024 * 1024, // 1MB buffer
+          env: {
+            ...process.env,
+            PATH: `/opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:${process.env.PATH || ''}`
+          },
+          stdio: ['pipe', 'pipe', 'pipe'] // Capture both stdout and stderr
+        }).toString().trim();
+        
+        console.log(`Direct execution successful, output length: ${output.length}`);
+      } catch (directError: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)